lib/experimental-default-theme.json

```diff
@@ -160,6 +160,42 @@
 						"slug": "huge",
 						"size": 42
 					}
+				],
+				"fontFamilies": [
+					{
+						"fontFamily": "-apple-system,BlinkMacSystemFont,\"Segoe UI\",Roboto,Oxygen-Sans,Ubuntu,Cantarell,\"Helvetica Neue\",sans-serif",
```

Comment:
Nitpick: can this be `family` instead of `fontFamily`? This would be more coherent with what we do with font-size (we use `size` there).